gramps/gen/plug/_gramplet.py

```diff
@@ -127,15 +128,20 @@ def get_active(self, nav_type):
         """
         Return the handle of the active object for the given navigation type.
         """
-        return self.uistate.get_active(nav_type, self.nav_group)
+        handle = self.uistate.get_active(nav_type, self.nav_group)
+        if handle and nav_type in self.dbstate.db.get_table_names() and \
```

Comment:
The `has_handle` method is not actually part of the public API and should be made protected.  This is currently under review - see bug [#9541](https://gramps-project.org/bugs/view.php?id=9541).

We need to discuss this further on the list.